docs/common/community/talks.md

```diff
@@ -1,6 +1,6 @@
 ---
 url: talks
-weight: 300
+weight: 400
```

Comment:
why is contributing before talks? I think blogs (200) and talks should be next to each other.

Reply:
yeah that order sounds good to me